### DISCUSSION TITLE
how could i add a new command category

### DISCUSSION THREAD

**Discussion by Derei-Ramos** [Androz2091/AtlantaBot#483] [Q&A] [answered]
can someone help me as i could add a new category in the command folder (i wanted to initially add the "minecraft" folder and do other commands within it)

**Comment by Androz2091** [comment, marked as answer]
* Create a new folder named `minecraft`
* Copy the `commands/General/ping.js` file into the created folder
* Change all the "ping" occurences to the name of your command
* Remove the content of the `run()` function to replace it with your own.
* Restart the bot
* You're done, your command will now appear in the help command!
